src/Microsoft.ML/Models/OnnxConverter.cs

```diff
@@ -10,9 +10,50 @@ namespace Microsoft.ML.Models
     public sealed partial class OnnxConverter
     {
         /// <summary>
-        /// Converts the model to ONNX format.
+        ///
```

Comment:
>/// [](start = 8, length = 3)

Would a link to ONNX project itself be appropriate, to motivate why someone might care?